### DISCUSSION TITLE
difference between states and saves

### DISCUSSION THREAD

**Discussion by noxtras** [Swordfish90/Lemuroid#549] [Q&A]
I recently 'backed up' my saves folder, and reinstalled Lemuroid from the play store (it was from F-droid before).. copied back the saves folder and to my surprise, all my saves are lost... I just noticed that saves, is actually called 'save state'... so what's the difference between them, and what did I backup?
I assume my saves are lost, right?:)

**Comment by xalamenders** [comment]
Save state means you save the game state (ram i guess?) without save spoint/autosave etc. If you load it, you continue at the point you save the game state (say, even in the mids of fight against ultima weapon). 

While save game is how the game save progress, the traditional save point.

Correct me if im wrong, I'm not tech savvy enough to explain...

Your save might not lost if you could load the save state.
